### PR TITLE
Add docs and fix major ticks positions

### LINKED ISSUE #3: Fix major ticks position during zoom

### PR DESCRIPTION
**Pull request by ondrej-tucek**
Resolve #3

### CODE REVIEW

File: lib/chart/gauge.ex [hunk 1]
```diff
@@ -1,11 +1,20 @@
 defmodule Chart.Gauge do
   @moduledoc """
-  Gauge graph
+
+  ![Gauge](docs/gauge.png)
+
+  An appearance of gauge graph can be change via `config_keywords` (see below `setup` function)
```
Comment (s-m-i-t-a) [inline lib/chart/gauge.ex]: [Make reference to function as html anchor.](https://hexdocs.pm/elixir/writing-documentation.html#recommendations)

File: lib/chart/gauge/settings.ex [hunk 2]
```diff
@@ -35,18 +35,18 @@ defmodule Chart.Gauge.Settings do
             key_guard(config, :major_ticks_count, 7, &Validators.validate_major_ticks_count/1),
           gap: key_guard(config, :major_ticks_gap, 0, &Validators.validate_number/1),
           length:
-            key_guard(config, :major_ticks_length, 7, &Validators.validate_positive_number/1)
+            key_guard(config, :major_ticks_length, 5, &Validators.validate_positive_number/1)
         }
-        |> set_translate(settings.gauge_center)
+        |> set_translate(settings.gauge_center, settings.gauge_radius)
```
Comment (s-m-i-t-a) [inline lib/chart/gauge/settings.ex]: Pass only `settings` struct. The function will be simple.

**Review by s-m-i-t-a — CHANGES_REQUESTED** [review]

File: lib/chart/gauge/settings.ex (hunk 2 above)
Comment (ondrej-tucek) [reply lib/chart/gauge/settings.ex]: I think it depends on point of view. Calling `set_translate(settings)` function will be simple, but her definition already won't + necessary `alias ...Settings` in this submodule. Compare
` defp set_translate(major_ticks, {cx, cy}, {rx, _ry}) do`
with
` defp set_translate(major_ticks, %Settings{gauge_center: {cx, cy}, gauge_radius: {rx, ry}},) do`

File: lib/chart/gauge.ex (hunk 1 above)
Comment (ondrej-tucek) [reply lib/chart/gauge.ex]: thx, I always forget it

File: lib/chart/gauge/settings.ex (hunk 2 above)
Comment (s-m-i-t-a) [reply lib/chart/gauge/settings.ex]: Not necessarily. Add for free _design by contract_ and follow a _crate pattern._

**Review by s-m-i-t-a — DISMISSED** [review]

**Review by s-m-i-t-a — APPROVED** [review]

**Pull request merged**